jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/actions/CodeWhispererRecommendationAction.kt

```diff
@@ -19,11 +20,14 @@ class CodeWhispererRecommendationAction : AnAction(message("codewhisperer.trigge
     }
 
     override fun actionPerformed(e: AnActionEvent) {
+        val latencyContext = LatencyContext()
+        latencyContext.codewhispererPreprocessingStart = System.nanoTime()
+        latencyContext.codewhispererEndToEndStart = System.nanoTime()
```

Comment:
since we end up converting to ms at the end, why not use `currentTimeMillis` directly?

Reply:
I remembered we encountered some issues when using `currentTimeMillis` long time ago but forgot the exact details. Otherwise it's totally fine to use it.